app/components/candidate_interface/complete_section_component.html.erb

```diff
@@ -3,19 +3,14 @@
 
   <%= render(review_component) %>
 
-  <%= f.govuk_radio_buttons_fieldset :completed, classes: 'govuk-!-width-two-thirds ', legend: { text: t('application_form.completed_question'), size: 'm' } do %>
-    <% if hint_text %>
-      <% if hint_text.is_a?(Array) %>
-        <% hint_text.each do |paragraph| %>
-          <div class="govuk-hint"><%= paragraph %></div>
-        <% end %>
-      <% else %>
-        <div class="govuk-hint"><%= hint_text %></div>
-      <% end %>
-    <% end %>
-    <%= f.govuk_radio_button :completed, true, label: { text: complete_or_reviewed_radio_button_label }, link_errors: true %>
-    <%= f.govuk_radio_button :completed, false, label: { text: t('application_form.incomplete_radio') } %>
-  <% end %>
+  <div class="govuk-!-width-two-thirds">
```

Comment:
The `govuk_collection_radio_buttons` accepts classes but unfortunately, these styles are not applied to any hints. Would wrapping in a `<div>` suffice?

![Uploading govuk_collection_radio_buttons.png…]()

Reply:
Wonder if this is an issue that needs to be filed upstream? @peteryates

Reply:
For now, a `div` around the `govuk_collection_radio_buttons` with the width override class will work.

Reply:
> Wonder if this is an issue that needs to be filed upstream? @peteryates

Something like `hint: { text: hint_text, classes: 'foo bar boz' } ` could be nice? 

Happy to raise a fix if it's a goer @peteryates

Reply:
I was wondering more if classes added should apply to `govuk-form-group` and not `govuk-radios`? I think the GDS Nunjucks templates allow you to do both.

Reply:
So the form builder has allowed custom hint content for some time via a proc. When a proc is provided it'll use a `div` instead of a `span`. Example:

```erb
<%= form.govuk_text_field :name, hint: -> do %>
  <section style="outline: 2px dashed palevioletred; padding: .5em 1em">
    <h4>Look!</h4>
    <p>Multi-paragraph</p>
  </section>
<% end %>
```

Renders:

![Screenshot from 2021-04-27 18-00-43](https://user-images.githubusercontent.com/128088/116282321-855d3f80-a782-11eb-8dcf-454ac04dd1b5.png)


I did notice that the guide's a little out of date, I'll update it tomorrow.

Reply:
Great thanks for that @peteryates!